module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -571,17 +611,16 @@ public function patronLogin($username, $password)
             throw new ILSException('Problem with Koha REST API.');
         }
 
-        $result = $result['data'];
         return [
-            'id' => $result['patron_id'],
-            'firstname' => $result['firstname'],
-            'lastname' => $result['surname'],
+            'id' => $data['patron_id'],
+            'firstname' => $data['firstname'],
+            'lastname' => $data['surname'],
             'cat_username' => $username,
-            'cat_password' => $password,
-            'email' => $result['email'],
+            'cat_password' => $password || '',
```

Comment:
Should this be `??` rather than `||`? I think we want to default to `''` if `$password` is null, but we might not want to for other falsy values -- i.e. what if your password is `00000000`?

Reply:
I think you've got a good point, well caught - I'll have a quick test and push up a commit when I'm happy